pkgs/applications/networking/instant-messengers/tangram/default.nix

```diff
@@ -27,15 +45,35 @@ stdenv.mkDerivation rec {
     ninja
     pkg-config
     python3
+    blueprint-compiler
     wrapGAppsHook
```

Comment:
Why keeping wrapGAppsHook here only to set dontWrapGApps later? (Or it has some side effects that I don't know)

Reply:
wrapGApp is used in postFixup. However I think this can be dropped by moving the sed to preFixup.